index.html

```diff
@@ -1,5 +1,3 @@
----
```

Comment:
Don't remove these lines @gautamjajoo they are needed for jekyll, learn more about jekyll and gh-pages see here: https://help.github.com/categories/github-pages-basics/